scipy/io/netcdf.py

```diff
@@ -989,4 +994,4 @@ def __setitem__(self, index, data):
 
 
 NetCDFFile = netcdf_file
-NetCDFVariable = netcdf_variable
+NetCDFVariable = netcdf_variable
```

Comment:
Thanks for the feedback, Andy.
I'm new to this stuff so I didn't know about this standard practice until now, thanks for informing me :)
By any means it appears that the issue this pull request meant to solve has already been solved by a past patch without resolving the issue.
Thanks again, though!